target/linux/ath79/image/tiny-tp-link.mk

```diff
@@ -183,8 +183,10 @@ TARGET_DEVICES += tplink_tl-wr841-v11
 define Device/tplink_tl-wr941-v2
   $(Device/tplink-4m)
   ATH_SOC := ar9132
-  DEVICE_MODEL := TL-WR941N/ND
+  DEVICE_MODEL := TL-WR941ND
   DEVICE_VARIANT := v2/v3
+  DEVICE_ALT0_MODEL := TL-WR941N
+  DEVICE_ALT0_VARIANT := v2/v3
   TPLINK_HWID := 0x09410002
   TPLINK_HWREV := 2
```

Comment:
BTW: Is this needed? It's the only device in tiny-tp-link.mk having it defined.

Reply:
It's required. Otherwise TP-Link's OEM firmware will reject it.